orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -134,7 +134,7 @@ class CreateWebhookTask implements RetryableTask {
       }
       return new TaskResult(ExecutionStatus.SUCCEEDED, outputsDeprecated + outputs)
     } else {
-      outputs.webhook << [error: "The request did not return a 2xx/3xx status"]
+      outputs.webhook << [error: "The request returned an HTTP status code that couldn't be processed"]
```

Comment:
Not sure I follow what you mean. Since some error codes can be allowed now (e.g. when you specify to fail fast on 503, that will go through the code path here https://github.com/spinnaker/orca/pull/2612/files/e52f946516716af22dc36ba77ce63a87cbe7c5c8#diff-e737f2c3e6b9c31027d9d0670960c694R69 . 
Perhaps the right message is "The webhook request failed" since it's accurate, what do you think?